src/traverse.js

```diff
@@ -0,0 +1,27 @@
+import { CID } from './index.js'
+import { Block } from './block.js'
+import { base58btc } from 'multiformats/bases/base58'
+
+/**
+ * @template T
+ * @param {Object} options
+ * @param {CID} options.cid
```

Comment:
@rvagg importing `CID` and `Block` for these `@param`s but not using them outside of this (JSDoc?) comment, means that running the standard linter (`npm run lint`) is not happy with the imports of `CID` and `Block`. Any ideas how to get around this?

Reply:
Import them with a typedef to use them in JSDoc type annotations:

```js
/**
 * @typedef {import('../cid.js').CID} CID
 */
```

Reply:
Thanks @achingbrain that answers my question!